project/src/main/webapp/script.js

```diff
@@ -68,5 +79,42 @@ function createPlaceElement(place) {
 function tryAgain() {
   document.getElementById('query-form').style.display = 'block';
   document.getElementById('results').style.display = 'none';
+  document.getElementById('waiting-message').style.display = 'block'
   document.getElementById('place').innerHTML = '';
 }
+
+
+/**
+ *  Creates a map and adds it to the page.
+ */
+function createMap() {
+  const USER_LOCATION = { lat: 32.080576, lng: 34.780641 }; //TODO(M1): change to user's location
+  var map = new google.maps.Map(document.getElementById('map-container'), {
+    center: USER_LOCATION,
+    zoom: 12,
+  });
+  return map;
+}
+
+/**
+ * Adds to the map a place's marker.
+ */
+function addPlaceMarker(map, place) {
+  const marker = new google.maps.Marker({
+      title: place.name,
+      position: place.location,
+      description: place.name.link(place.websiteUrl),
+      map: map
+  });
+  if (place.websiteUrl) {
+    const infoWindow = new google.maps.InfoWindow({content: marker.description});
+    marker.addListener('click', () => {
+        infoWindow.open(map, marker);
+    });
+  }
+  // Zoom when clicking on marker
+  google.maps.event.addListener(marker,'click', function() {
```

Comment:
Can also use "() => {}" insted of "function() {}". I personally like it better, and I'm not sure if one is preferred over the other.

Reply:
right, I changed it like you suggested so it will be in the same format as other eventlisteners in the function.